libs/dsb-transport-core/src/transport.interface.ts

```diff
@@ -5,7 +5,13 @@ export interface ITransport {
     //TODO: consider moving clientId to generic type like NATSJetstreamOptions if clientId is NATSJS specific
     pull(fqcn: string, clientId: string, amount: number): Promise<Message[]>;
     publish(fqcn: string, payload: string): Promise<string>;
-    createChannel(fqcn: string): Promise<string>;
+    createChannel(fqcn: string, metadata?: any): Promise<string>;
```

Comment:
okay for now but I'd specify the type here - never know who will call this and don't want to populate the address book with garbage :)

Reply:
yeah but i was not sure to add address book interface here and make this package dependent on that. but i can re-introduce metadata type here as well